docs/markdown/Java and Scala/kotlin.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Kotlin"
+slug: "kotlin"
+excerpt: "Kotlin support for Pants."
+hidden: false
+---
+
+> 🚧 Kotlin support is alpha stage
+>
+> Kotlin support in Pants is still under active development, but currently supports compilation and testing. It has
+> been tested with Kotlin v1.6.20.
+>
+> Please share feedback for what you need to use Pants with your Kotlin project by either
+> [opening a GitHub issue](https://github.com/pantsbuild/pants/issues/new/choose)
+> or [joining our Slack](doc:community)!
+
+> 📘 Example Kotlin repository
+>
+> Check out [github.com/pantsbuild/example-kotlin](https://github.com/pantsbuild/example-kotlin) to try a
+> sample Pants project with Kotlin support.
+
+# Overview
+
+[Kotlin](https://kotlinlang.org/) is a programming language from Jetbrains that runs on the JVM and certain other
+platforms. The Kotlin backend in Pants supports compilation, testing, and linting of
+[Kotlin code for the JVM](https://kotlinlang.org/docs/server-overview.html). (The other Kotlin platforms including
+[Kotlin Multiplatform Mobile](https://kotlinlang.org/docs/multiplatform.html) and
+[Kotlin/JS](https://kotlinlang.org/docs/js-overview.html) are not currently supported, nor are there currently
+any plans to do so.)
+
+# Initial Setup
+
+First, activate the Kotlin backend in `pants.toml` plus the `ktlint` backend if you would like to use
+[`ktlint`](https://ktlint.github.io/) for code formatting and linting:
+
+```toml pants.toml
+[GLOBAL]
+backend_packages = [
+  "pants.backend.experimental.kotlin",
+  
+  # Activate the following backend if you want to use `ktlint` for code formatting and linting.
+  "pants.backend.experimental.kotlin.lint.ktlint",
+]
+```
+
+## Choosing JDK and Kotlin versions
+
+Pants supports choosing the JDK and Kotlin versions per target in your repository, but to reduce the amount of
```

Comment:
Nit to make it simpler to read

```suggestion
Pants supports choosing the JDK and Kotlin versions per target in your repository. However, to reduce the amount of
```

Reply:
Made edit but moved "however" after the "to" clause.